### PR TITLE
RATEST-115:Un Ignore SearchActiveVisitTest

### PR DESCRIPTION
**Pull request by sherrif10**
https://issues.openmrs.org/browse/RATEST-115
cc @k-joseph

### CODE REVIEW

**Comment by HerbertYiga** [comment]
@sherrif10  thanks sharif,did you make any work around to fix this https://talk.openmrs.org/t/session-id-is-null-using-webdriver-after-calling-quit/32994/4

**Comment by sherrif10** [comment]
Nope, ideally that error on talk still exists, For these tests that dont need some logic under automations workflow will pass , but that kind of test in AddForm and other related work that fails will return that error of session ID

File: ui-tests/src/test/java/org/openmrs/reference/SearchActiveVisitsTest.java [hunk 1]
```diff
@@ -31,7 +30,7 @@ public void setup() {
         new TestData.TestVisit(patient.uuid, TestData.getAVisitType(), getLocationUuid(homePage)).create();
     }
 
-    @Ignore
+
```
Comment (kdaud) [inline ui-tests/src/test/java/org/openmrs/reference/SearchActiveVisitsTest.java:33]: @sherrif10 did you just forget to add @Test annotation to this test.

Comment (sherrif10) [reply ui-tests/src/test/java/org/openmrs/reference/SearchActiveVisitsTest.java:33]: Thanks @kdaud Resolved it

**Comment by sherrif10** [comment]
cc @k-joseph  is this ready to be merged

**Review by sherrif10 — COMMENTED** [review]
cc @k-joseph

**Comment by sherrif10** [comment]
Hello @kaweesi  could we merge this please

**Comment by kdaud** [comment]
Have tested the changes locally but there could be some more work to do. Do you mind testing these changes with _chromeheadless_  instance

**Comment by sherrif10** [comment]
There is no more work to be done here i guess, i tested this work on both chrome driver and firefox driver  and all was well unless am getting your point mistakenly thanks

**Comment by kdaud** [comment]
> There is no more work to be done here i guess

@sherrif10 you may be right. While testing your changes I realised there could be some little work to be done. If you do not mind test these changes once again with _chromeheadless_ set to _true_. Some thing like headless=true

**Comment by sherrif10** [comment]
could you please share those changes you thing are needed to we added, otherwise it works as expected on headless =true

File: ui-tests/src/test/java/org/openmrs/reference/SearchActiveVisitsTest.java [hunk 2]
```diff
@@ -31,7 +31,7 @@ public void setup() {
         new TestData.TestVisit(patient.uuid, TestData.getAVisitType(), getLocationUuid(homePage)).create();
     }
 
-    @Ignore
+    @Test
```
Comment (kaweesi) [inline ui-tests/src/test/java/org/openmrs/reference/SearchActiveVisitsTest.java:34]: Add `@Category(BuildTests.class)` too

Comment (sherrif10) [reply ui-tests/src/test/java/org/openmrs/reference/SearchActiveVisitsTest.java:34]: Thanks @kaweesi , this is already been added here thanks, it passess on both drivers

**Comment by kaweesi** [comment]
Please check through and fix https://ci.openmrs.org/browse/REFAPP-UI-FRAMEWORK-102/test/case/214401051

**Pull request merged**